src/app/form-generator/components/node-field/node-field.scss

```diff
@@ -30,3 +38,47 @@ label {
     align-items: center;
     transition: width 0.2s;
 }
+
+.form {
+    display: flex;
+    right: 0;
+}
+
+gtx-contents-list-item {
+    width: 100%;
+    border: none;
+}
+
+.title {
+    width: 150px;
+    margin-left: -1%;
+}
+
+.schemaName {
+    font-size: 12px;
+    display: flex;
```

Comment:
Remove `display: flex`. It has no effect

Reply:
We need `display: flex` here to show the border right.